acronym-mapper/src/main/java/de/intension/acronym/AcronymMapper.java

```diff
@@ -83,12 +101,13 @@ public void updateBrokeredUser(KeycloakSession session, RealmModel realm, UserMo
 
     private void updateUser(UserModel user, IdentityProviderMapperModel mapperModel) {
         var attribute = mapperModel.getConfig().getOrDefault(ATTRIBUTE, ATTRIBUTE_DEFAULT_VALUE);
+        String modifier = mapperModel.getConfig().getOrDefault(MODIFIER, MODIFIER_LOWER_CASE);
```

Comment:
Perfect, this part is very important for migration to this new mapper - when upgrading an environment clients with this mapper already configured will probably return `null` here :+1:

Reply:
Thank you very much for the flowers!